src/sass/patterns/_form-elements.scss

```diff
@@ -10,9 +10,67 @@
   grid-template-columns: 1fr auto;
 }
 
-[data-form='task-capture'] {
+.metecho-nested-checkboxes {
+  // line up with above label
+  // add grid + (faux checkbox + label margin)
+  padding-left: var(
+    --nested-checkboxes-padding-left,
+    calc($square-icon-x-small-boundary + $spacing-large)
+  );
+}
+
+.form-grid-three {
+  display: grid;
+  gap: $spacing-x-small;
+  grid-template-columns: 43% 43% 1fr;
+
+  // allow multi-line checkbox labels to line up on left
+  .slds-checkbox__label {
+    display: grid;
+    grid-template-columns: min-content 1fr;
+  }
+
+  // when using grid on wrapper,
+  // vertically align faux checkbox with with label
+  .slds-checkbox_faux {
+    margin-top: 0.35rem;
+  }
+
+  .slds-accordion__content & {
+    margin-left: $square-icon-x-small-boundary;
+  }
+
+  .metecho-nested-checkboxes {
+    --nested-checkboxes-padding-left: #{$spacing-large};
+  }
+}
+
+[data-form='task-retrieve-changes'] {
   display: grid;
   height: 100%;
+  overflow: inherit;
+
+  .slds-card {
+    display: flex;
+    flex-direction: column;
+    height: inherit;
+    overflow: inherit;
+  }
+
+  .metecho-panel-container {
+    height: inherit;
+  }
+
+  .slds-card__body {
+    height: inherit;
+    overflow: inherit;
+
+    > div {
+      display: flex;
+      flex-direction: column;
+      height: inherit;
+    }
+  }
```

Comment:
@mirisuzanne am I approaching this in the hardest way possible?

Reply:
I don't know exactly what you're trying to accomplish - but that looks like a reasonable override of some unwanted styles, and then a bit of new flex behavior. I would consider using explicit values or `unset` instead of `inherit` for heights and overflow - just to make it more clear what you're trying to achieve?